roles/caliopen/tasks/main.yaml

```diff
@@ -83,9 +84,16 @@
     host: 0.0.0.0
     port: 9042
     delay: 30
+  when: stage == 'false'
+
+- name: pull docker services images
```

Comment:
Waiting for cassandra to be up on port on 0.0.0.0:9042 is not a option for staging, atm we just pull the images to make some time before starting the setup. Not the best option.